bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/driver/Port.java

```diff
@@ -300,21 +300,23 @@ public void run() {
         }
 
         private void processMessages() {
-            try {
-                // must call processData() until we get a null pointer back
-                for (Msg m = msgFactory.processData(); m != null; m = msgFactory.processData()) {
-                    toAllListeners(m);
-                    notifyWriter(m);
-                }
-            } catch (IOException e) {
-                // got bad data from modem,
-                // unblock those waiting for ack
-                logger.warn("bad data received: {}", e.getMessage());
```

Comment:
This was removed because it was getting logged twice just before the `IOException` is raised. That's why the `bad data received` prefix was moved to the other log statement.